ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -724,6 +833,8 @@ export default class AdvancedTab extends PureComponent {
     return (
       <div className="settings-page__body">
         {warning ? <div className="settings-tab__error">{warning}</div> : null}
+        {this.renderUserDataBackup()}
+        {this.renderRestoreUserData()}
```

Comment:
I am not sure whether `renderUserDataBackup` and `renderRestoreUserData` should be the first entry on the Advanced Tab since the 3box toggle is the 4th entry from the end. Does it make sense to move them closer to the 3box toggle itself?

Reply:
@danjm suggested we put it at the top.

Reply:
We can ignore my previous suggestion and follow @NiranjanaBinoy's suggestion here